src/util/ImageUtil.js

```diff
@@ -1,12 +1,12 @@
 L.ImageUtil = {
 
-  getCmPerPixel(overlay) {
-    var map = overlay._map;
+  getCmPerPixel:((overlay)=> {
+    let map = overlay._map;
```

Comment:
Here I think it could be const perhaps? And on line 6 as well, if I understand right! 

Thank you for your help!!

Reply:
Hi @kabuur would you mind making this change and then hopefully we can merge this! Thank you!
```suggestion
    const map = overlay._map;
```